### ISSUE TITLE
[Profiler] Add detailed request time desc. (dns lookup, ...)

### ISSUE THREAD

**Issue by florianpreusner** [8p/EightPointsGuzzleBundle#224]
Based on #210:

More detailed request time description: dns lookup time, connection time, etc..

**Comment by rpkamp** [comment]
This sounds awesome, but where could we get that information? I've searched for a bit, but couldn't find anything on how to retrieve these timings from Guzzle.

**Comment by scion4581** [comment]
Yeah, but how to access request time in symfony log middleware, for example, to put in log context...
